docs/Security-Events-Preview-Configuration.md

```diff
@@ -9,7 +9,7 @@ Collection of the following 3rd party security log types is supported:
 ##Configuration summary
 1. Install and onboard the OMS Agent for Linux
 2. Configure Syslog forwarding to send the required logs to the agent on UDP port 25226
```

Comment:
Add a step here about acquiring the workspace ID for the workspace that they want the logs to be sent to.